src/main/java/github/scarsz/discordsrv/commands/CommandInvite.java

```diff
@@ -0,0 +1,21 @@
+package github.scarsz.discordsrv.commands;
+
+import github.scarsz.discordsrv.util.DiscordUtil;
+import net.dv8tion.jda.api.Permission;
+import org.bukkit.ChatColor;
+import org.bukkit.command.CommandSender;
+
+public class CommandInvite {
+    @Command(commandNames = { "invite" },
+            helpMessage = "Generates an invite code for the bot",
+            permission = "discordsrv.invite"
+    )
+    public static void execute(CommandSender sender, String[] args) {
+        if (DiscordUtil.getJda() != null) {
+            String invite = DiscordUtil.getJda().getInviteUrl(Permission.ADMINISTRATOR);
```

Comment:
This should really be avoided imo.
Someone just needs to get the token to mess up the entire Discord server.

Just give back an invite with no permissions set (Similar to https://scarsz.me/authorize)
```suggestion
            String invite = DiscordUtil.getJda().getInviteUrl();
```

Reply:
This is a whole other conversation, the Installation guide already instructs users to create a role with Administrator permissions; or individual permissions for _advanced users_.

Reply:
Then the guide should be updated.
Like I said should we stop with this habit of giving out admins to bots.
If people won't spend time to setup perms should they also not run srvers.

Reply:
The Installation guide says (in a condensed format) `Add the Administrator permission to the bot. If you're experienced, here's a list of permissions the bot needs`. It's not our job to dictate what others do with their discord servers. I do support your commit suggestion, though.